docs/api/index.md

```diff
@@ -19,15 +19,27 @@ CiviCRM, using the API explorer and the API parameter list.
 
 For help creating your own API custom calls, see [civix generate:api](/extensions/civix.md#generate-api)
 
+## Versions
+
+CiviCRM has two different versions of the API. Version 3 is considered the current stable version and Version 4 is the currently worked on version.
```

Comment:
I don't think this assertion creates quite the right impression and will quickly grow stale. If you think some kind of comparison/trade-off is really needed, then maybe point out that (a) v3 has been around longer and has more entities (b) v4 has cleaner/more consistent design.